README.md

```diff
@@ -340,3 +341,28 @@ services:
 volumes:
   redis:
 ```
+
+### Search
+
+- [ELK Stack: Elasticsearch, Logstash, Kibana](#ELK)
+
+#### ELK
+```yaml
+
+# Required on Linux:
+# sudo sysctl -w vm.max_map_count=262144
+# sudo sh -c 'echo "vm.max_map_count=262144" >> /etc/sysctl.conf'
+
+services:
+  elk:
+    image: sebp/elk:683
```

Comment:
Why not https://github.com/deviantony/docker-elk ?

Reply:
Never used it. I'm using sebp/elk in development since a few weeks ago and I'm happy with it.